data/src/main/java/org/apache/iceberg/data/orc/GenericOrcWriter.java

```diff
@@ -109,6 +112,10 @@ private WriteBuilder() {
               iPrimitive, primitive));
       }
     }
+
+    private int getFieldId(TypeDescription typeDescription) {
```

Comment:
nit: the private method feels redundant since the body is also just one line

Reply:
I personally think it's easier to read by segregating the abstract logic of what's doing here from the actual underlying implementation so I created the helper; do you feel strong about this? I'll see if other people have the same comment?

Reply:
I agree with @jackye1995. I probably wouldn't separate this out into its own method.